__tests__/pages/_document.test.js

```diff
@@ -1,9 +1,16 @@
 import '@testing-library/jest-dom'
 import MyDocument from '../../pages/_document'
+import * as utils from '../../lib/Utils.js'
```

Comment:
Suggestion: We typically import only the functions we need, not the whole library
```suggestion
import { generateNonce } from '../../lib/Utils'
```